src/cargo/util/context/mod.rs

```diff
@@ -2860,26 +2860,6 @@ where
             formatter.write_str("a string (\"auto\" or \"never\") or a table")
         }
 
-        fn visit_str<E>(self, s: &str) -> Result<Self::Value, E>
```

Comment:
should `fn expecting` be updated?

Reply:
Updated and anyway it is eventually removed.
Also rebased onto master.

I am going to hit the merge button.